pxemgr/config.go

```diff
@@ -53,9 +53,10 @@ type Profile struct {
 }
 
 type Network struct {
-	Interface      string
-	BindAddr       string `yaml:"bind_addr"`
-	BootstrapRange struct {
+	Interface        string
+	MachineInterface string `yaml:"machine_interface"`
```

Comment:
Wanted to find this :heart:

Reply:
I think it makes sense to rename the `interface` to `pxe_interface`  for easier understanding.